src/components/pageNav/index.jsx

```diff
@@ -33,13 +55,35 @@ class PageNav extends React.PureComponent {
     }));
   };
 
+  handleShowCompletedAlert = () => {
```

Comment:
the "Completed" language prefixing all of the variable and method names related to this alert makes sense to me.  For further clarity please update the other alert's variable and method names with something equally as distinct.  As is it reads now it's like there is some generic all purpose alert getting toggled, and then this new highly specialize alert doing its own thing, which isn't really true to what's goin on here.